### PR TITLE
Change Gastronomicon build link

### PR DESCRIPTION
**Pull request by SchnTgaiSpock**
Switching to blob builds

### CODE REVIEW

**Review by Phoenix-Starlight — APPROVED** [review]
LGTM. The backslash doesn't seem to make a difference though. Perhaps remove the backslash.

**Review by svr333 — APPROVED** [review]

File: pages/Addons.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Here is a list of all Slimefun Addons (that we currently know of). You can insta
 - [:heart: Community Addons](#heart-community-addons)
 - [:gift: Third-Party Plugins](#gift-third-party-plugins)
 - [:trophy: Addon Jams](#trophy-addon-jams)
-- [:black_nib: Create your own addons!](#black_nib-create-your-own-addons)
+- [:black\_nib: Create your own addons!](#black_nib-create-your-own-addons)
```
Comment (WalshyDev) [inline pages/Addons.md:9]:
```suggestion
- [:black_nib: Create your own addons!](#black_nib-create-your-own-addons)
```

**Review by WalshyDev — APPROVED** [review]

**Comment by SchnTgaiSpock** [comment]
> LGTM. The backslash doesn't seem to make a difference though. Perhaps remove the backslash.

sorry, just remembered this happened last time too. no idea why vscode does this

**Comment by Boomer-1** [comment]
Just an fyi - the link for Gastronomicon was already updated to blob.builds with pr #272

**Comment by SchnTgaiSpock** [comment]
ah ty

**Pull request closed without merging**